src/components/Toc.vue

```diff
@@ -60,7 +60,7 @@ export default {
       onClick(e);
     });
 
-    // Change mask postion on scroll
+    // Change mask position on scroll
```

Comment:
@Flamenco : Excellent attention to detail for the comment spelling correction.

Reply:
@waellerbe, I think this might be a **breaking change**.  Good thing it has not been merged...